src/tours/Steps/index.tsx

```diff
@@ -0,0 +1,9 @@
+import getVirtualStudySteps, { virtualStudyId } from './VirtualStudy';
```

Comment:
i'm curious what importing and then exporting from this index file achieves.  i know we do this in packages so that you can import from the package but what here?

Reply:
I want to organize the import and export of methods and variables related to steps all within a single file. 
This is just my personal preference, and deleting this file is entirely feasible. 
How do you think about?

Reply:
i see.  that makes sense.